### PR TITLE
Modified model and form to not allow negative input

### PR DESCRIPTION
**Pull request by hurt1030**

### CODE REVIEW

**Comment by shaneallgeier** [comment]
Could you solve the merge conflicts? I like to see the "final product" whenever I do a review

**Comment by hurt1030** [comment]
@shaneallgeier merge conflicts have been resolved

File: rim/migrations/0012_auto_20200311_1052.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    operations = [
+        migrations.AlterField(
+            model_name='equipment',
+            name='count',
+            field=models.IntegerField(blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)]),
+        ),
+        migrations.AlterField(
+            model_name='equipment',
+            name='purchase_price',
+            field=models.DecimalField(blank=True, decimal_places=2, max_digits=5, null=True, validators=[django.core.validators.MinValueValidator(0)]),
```
Comment (shaneallgeier) [inline rim/migrations/0012_auto_20200311_1052.py]: This line is inadvertently reverting the changes from #55. Just manually edit this line to be `decimal_places=2, max_digits=7` instead and it should be good to go.

Comment (hurt1030) [reply rim/migrations/0012_auto_20200311_1052.py]: Done.

**Pull request merged**